### PR TITLE
Cg albumsdata update

### PR DESCRIPTION
**Pull request by Cgatewood55**
Checkout the updated albums data with tracks and and tracks array.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -176,36 +176,78 @@ var albumsArray = [
...
+    {
+        id:"t5",
+        title:"Killer Frog",
+        length: "4 min 15sec"
+    },
+    {
+        id:"t6",
+        title:"Return of the Frogs",
+        length: "2 min 30sec"
+    },
```
Comment (ghost) [inline main.js]: need to get rid of that comma

**Comment by ghost** [comment]
last comma after last curly bracket needs removal

**Comment by ledugani** [comment]
👍

**Comment by ghost** [comment]
👍

**Comment by ghost** [comment]
💯

**Pull request merged**